common-utils/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "common-utils"
-version = "0.1.4"
+version = "0.2.0"
```

Comment:
Just curious: why did you decide to do a minor version increase instead of a patch?

Reply:
this was because of the new module, so I was thinking new feature = minor and fixes = patch